server/supernova/CMakeLists.txt

```diff
@@ -106,22 +106,32 @@ else()
   endif()
 endif()
 
-if(APPLE OR MSYS)
+if((APPLE OR MSYS) AND (AUDIOAPI STREQUAL portaudio))
     find_package(Portaudio)
     if(NOT PORTAUDIO_FOUND)
         message(FATAL_ERROR "Portaudio missing")
     endif()
     target_compile_definitions(libsupernova PUBLIC PORTAUDIO_BACKEND ${PORTAUDIO_DEFINITIONS})
     target_include_directories(libsupernova PUBLIC ${PORTAUDIO_INCLUDE_DIRS})
     target_link_libraries(libsupernova ${PORTAUDIO_LIBRARIES})
-elseif (WIN32)
+elseif (WIN32 AND (AUDIOAPI STREQUAL portaudio))
     target_compile_definitions(libsupernova PUBLIC PORTAUDIO_BACKEND -DPORTAUDIO_BACKEND)
     target_include_directories(libsupernova PUBLIC ${PORTAUDIO_INCLUDE_DIRS})
     if(MSYS)
         target_link_libraries(libsupernova ${PORTAUDIO_LIBRARIES})
     else()
         target_link_libraries(libsupernova portaudio)
     endif()
+elseif (WIN32)
+	find_package(Jack)
+	if(NOT JACK_FOUND)
+		message(FATAL_ERROR "Jack selected as audio API, but development files not found")
+	endif()
+	if (JACK)
+        target_compile_definitions(libsupernova PUBLIC JACK_BACKEND)
+        target_link_libraries(libsupernova ${JACK})
+        target_include_directories(libsupernova PUBLIC ${JACK_INCLUDE_DIRS})
+    endif()
```

Comment:
this branch is exactly the same as the one below except for the fatal error check (which I agree should be added), so can you please merge them?

Reply:
not the same: find_library vs. find_package

Reply:
good point. still, they're very similar, could both use the same branch or is there a specific reason this one needs to be find_package (or a reason the other needs to be find_library?)

Reply:
find_package worked for me. find_library did not work.

Reply:
Is the "move if(JACK)....."  commit what you meant above?